docs/en/source/introduction.rst

```diff
@@ -5,7 +5,8 @@ Introduction
 Scorpio broker is a reference implementation of **NGSI-LD standard** specifications that are compliant to **ETSI standards**. Basically Scorpio broker is a core component of **FiWARE/IoT** platform wherein IoT data-driven by dynamic context is collected, processed, notified & stored/ingested with different application usage perspectives.
```

Comment:
1.Scorpio Broker is an NGSI-LD compliant context broker developed by NEC Laboratories Europe and NEC India as a part of the FIWARE platform. It implements the full NGSI-LD API as specified by the ETSI Industry Specification Group on cross cutting Context Information Management (ETSI ISG CIM). The NGSI-LD API enables the management, access and discovery of context information. Context information consists of entities (e.g., a building) and their properties (e.g., address and geographic location) and relationships (e.g., owner). Thus, Scorpio Broker enables applications and services to request context information – what they need, when they need it and how they need it.
The functionalities of the NGSI-LD API include:
•	Create, update, append and delete context information.
•	Query context information, including filtering, geographic scoping and paging.
•	Subscribe to changes in context information and receive asynchronous notifications.
•	Register and discover sources of context information, which allows building distributed and federated deployments.
2. Scorpio Broker is a FIWARE Generic Enabler. Therefore, it can be integrated as part of any platform “Powered by FIWARE”. FIWARE is a curated framework of open-source platform components which can be assembled with other third-party platform components to accelerate the development of Smart Solutions.

Reply:
Done